platform/native/contiki-main.c

```diff
@@ -242,6 +242,8 @@ main(int argc, char **argv)
 
   serial_line_init();
 
+  process_start(&sensors_process, NULL);
```

Comment:
I see your point but several platforms do it (cc2530dk, cc2538dk, cooja, econotag, remote, seedeye, sensinode, sky, srf06-cc25xx, wismote and z1), so the default is let applications start the sensors_process? If yes, those platforms should be fixed to avoid mistakes like this.